pages/portfolio.html

```diff
@@ -4,9 +4,41 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="/Users/miay/Developer/unit2-projects/personal-portfolio-site/styles/portfolio.css" rel="stylesheet">
   <title>Document</title>
 </head>
 <body>
-  
+  <header>
+    <h1>Projects</h1>
+  </header>
+  <section>
```

Comment:
This is a better use of `section` Since you have a project (which you provide as the header) and the details are the ssection content.